src/modules/gifmaker/GifRecorderOverlay/ScreenCapture.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GifRecorderOverlay.Util
+{
+    class ScreenCapture
```

Comment:
Is this supposed to be public or static?  Where will it be used and called from.

Reply:
I'll be setting it to public so it can be used in the coming ViewModel